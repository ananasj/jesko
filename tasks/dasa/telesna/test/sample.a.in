3
5 7 8
4
2 6 7 11
