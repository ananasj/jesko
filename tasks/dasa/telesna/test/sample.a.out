2 5 6 7 7 8 11
